pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -4,28 +4,37 @@ import (
 	"bufio"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
 	"strings"
 	"sync"
+	"time"
 
 	"github.com/hashicorp/hcl"
 	"github.com/spiffe/spire/proto/agent/workloadattestor"
 	"github.com/spiffe/spire/proto/common"
 	spi "github.com/spiffe/spire/proto/common/plugin"
 )
 
+const (
+	defaultMaxPollAttempts   = 15
```

Comment:
i'm not confident these are the right defaults... anybody have input?

Reply:
In the testing I did, even just 1 second was enough. I think what we are waiting on here is for docker container start event to be emitted on the event stream. I'd recommend really tight defaults here. Interval may be defined in ms instead of s. If I were to take a guess at this, I'd probably try defaulting MaxPoll to 5 and PollRetryInterval to ~300ms. Workloads are "supposed" to check back if they receive PermissionDenied error, which is what they'd get if we didn't find a match.

Reply:
Ok, i'll adjust accordingly. I'll have to change the config to accommodate a sub-second interval.